vignettes/baselinenowcast.Rmd

```diff
@@ -274,12 +282,60 @@ The reporting triangle we are applying it to must have the same `max_delay`
 as the delay estimate.
 
 ```{r}
-# point_reporting_square <- apply_delay( #nolint
-#   triangle_to_nowcast = triangle,#nolint
-#   delay_df#nolint
-# )#nolint
+point_reporting_square <- apply_delay(
+  triangle_to_nowcast = triangle,
+  delay_pmf = delay_df$pmf
+)
 ```
 
+We'll make a quick plot to compare the nowcasted confirmed cases through
+July 1, 2021, from the observations up until October 1, 2021.
+We'll compare this to the right-truncated data available up until July 1, 2021.
+```{r}
+final_data <- epinowcast::germany_covid19_hosp[location == "DE"][age_group == "00+"] |> # nolint
+  enw_filter_report_dates(latest_date = "2021-10-01") |>
+  enw_filter_reference_dates(
+    latest_date = "2021-07-01",
+    include_days = n_history - 1
+  ) |>
+  group_by(reference_date) |>
+  summarise(
+    total_confirmed = max(confirm)
```

Comment:
if you wanted you could pull this out of the preprocess object vs recalc (i.e have a at the time and final preprocess object)

Reply:
Going to leave for now but can adjust